rosmon_core/src/fmt_no_throw.h

```diff
@@ -0,0 +1,27 @@
+#include <fmt/format.h>
```

Comment:
Add `#pragma once` otherwise the headers will be included twice when its added transitively, causing issues